tests/PluginTest.php

```diff
@@ -101,6 +94,47 @@ public function dataProviderHandleHelp()
     }
 
     /**
+     * Tests private function getFlippedTable()
+     */
+    public function testGetFlippedTable()
+    {
+        // We have to set the private method accessible
+        $method = new ReflectionMethod(
+            'Phergie\Irc\Plugin\React\TableFlip\Plugin', 'getFlippedTable'
+        );
+        $method->setAccessible(TRUE);
+
+        // Test our Method
+        $this->assertEquals(
+            '(╯°□°）╯︵ ┻━┻ ', $method->invoke(new Plugin)
```

Comment:
how does this one receive a flipped table without the `$plugin->handleTableflipCommand($event, $queue);` being called?